frontend/src/pages/SubProjects/ProjectHistoryDrawer.js

```diff
@@ -33,20 +32,19 @@ function ProjectHistoryDrawer({
 
 function mapStateToProps(state) {
   return {
-    offset: state.getIn(["detailview", "offset"]),
-    limit: state.getIn(["detailview", "limit"]),
     doShow: state.getIn(["detailview", "showHistory"]),
     events: state.getIn(["detailview", "historyItems"]),
-    nEventsTotal: state.getIn(["detailview", "historyItemsCount"]),
-    hasMore: state.getIn(["detailview", "hasMoreHistory"]),
+    nEventsTotal: state.getIn(["detailview", "totalHistoryItemCount"]),
     isLoading: state.getIn(["detailview", "isHistoryLoading"]),
+    currentHistoryPage: state.getIn(["detailview", "currentHistoryPage"]),
+    lastHistoryPage: state.getIn(["detailview", "lastHistoryPage"]),
     getUserDisplayname: uid => state.getIn(["login", "userDisplayNameMap", uid]) || "Somebody"
   };
 }
 
 const mapDispatchToProps = {
   hideHistory,
-  fetchNextProjectHistory
+  fetchNextProjectHistoryPage
 };
 
 export default connect(mapStateToProps, mapDispatchToProps)(ProjectHistoryDrawer);
```

Comment:
a `toJS` is missing here in order to pass `events` as plan JS Object instead of passing an ImmutableJS object. Dumb components should not have ImmutableJS Objects: https://redux.js.org/recipes/using-immutablejs-with-redux#never-use-immutablejs-in-your-dumb-components

Reply:
changed view component, so it does not expect ImmutableJS objects anymore